stories/Templates/Articlepage/ArticlePage.jsx

```diff
@@ -33,24 +33,34 @@ const ArticlePage = ({
   footerData,
   headingLabel,
   featuredCardContent,
+
   languageswitcherData,
+  navigationData,
   rightNavigationData,
   leftNavigationData,
-  navigationData,
   locale,
+  getMegaMenuData,
```

Comment:
@Ashish-srij I do not see `getMegaMenuData` being used anywhere. Can you remove it if it is not required.

Reply:
done